carla/models/pipelining/steps.py

```diff
@@ -0,0 +1,68 @@
+def scaler(fitted_scaler, features, df):
```

Comment:
Can we rename it to `scale` and `encode` as they are functions and not properties? Also that makes it less confusing versus the scaler property of the MLModel

Reply:
Right, this would be a much better choice 👍